Calling/ClientApp/src/components/MediaGallery.tsx

```diff
@@ -26,10 +28,11 @@ export default (props: MediaGalleryProps): JSX.Element => {
   );
   const getMediaGalleryTilesForParticipants = (participants: RemoteParticipant[], userId: string, displayName: string) => {
     const remoteParticipantsMediaGalleryItems = participants.map((participant) => (
-      <div key={`${utils.getId(participant.identifier)}-tile`} className={mediaGalleryStyle}>
+       <div key={`${utils.getId(participant.identifier)}-tile`} className={mediaGalleryStyle}>
```

Comment:
this is the correct spacing or did we add a space here?

Reply:
Thanks, will fix.